mne/io/cnt/cnt.py

```diff
@@ -0,0 +1,295 @@
+        event_size = np.fromfile(fid, dtype='<i4', count=1)
+        if event_type == 1:
+            event_bytes = 8
+        elif event_type == 2:
+            event_bytes = 19
+        else:
+            raise IOError('Unexpected event size.')
+        n_events = event_size // event_bytes
+        events = list()
+        for i in range(n_events):
+            fid.seek(event_offset + 9 + i * event_bytes + 4)
+            offset = np.fromfile(fid, dtype='<i4', count=1)
+            events.append((offset - 900 - 75 * n_channels) //
+                          (n_channels * 2))
+        stim_channel = np.zeros(n_samples)
+
+        for event in events:
+            stim_channel[event - 1] = 1
+
+    info = _empty_info(cnt_info['sfreq'])
+    if lowpass_toggle == 1:
+        info['lowpass'] = highcutoff
+    if highpass_toggle == 1:
+        info['highpass'] = lowcutoff
+    info.update(filename=input_fname, meas_date=np.array([meas_date, 0]),
+                description=session_label, buffer_size_sec=10.)
+
+    coords = _topo_to_sphere(pos)
+    for idx, ch_name in enumerate(ch_names):
+        ch_coil = FIFF.FIFFV_COIL_EEG
+        ch_kind = FIFF.FIFFV_EEG_CH
+        loc = np.zeros(12)
+        loc[:3] = coords[idx]
+
+        chan_info = {'cal': cals[idx], 'logno': idx + 1, 'scanno': idx + 1,
+                     'range': 1.0, 'unit_mul': 0., 'ch_name': ch_name,
+                     'unit': FIFF.FIFF_UNIT_V,
+                     'coord_frame': FIFF.FIFFV_COORD_HEAD, 'loc': loc,
+                     'coil_type': ch_coil, 'kind': ch_kind}
+        chs.append(chan_info)
+
+    # Add the stim channel.
+    chan_info = {'cal': 1.0, 'logno': len(chs) + 1, 'scanno': len(chs) + 1,
+                 'range': 1.0, 'unit_mul': 0., 'ch_name': 'STI 014',
+                 'unit': FIFF.FIFF_UNIT_NONE,
+                 'coord_frame': FIFF.FIFFV_COORD_UNKNOWN, 'loc': np.zeros(12),
+                 'coil_type': FIFF.FIFFV_COIL_NONE, 'kind': FIFF.FIFFV_STIM_CH}
+    chs.append(chan_info)
+    baselines.append(0)  # For stim channel
+    cnt_info.update(baselines=np.array(baselines), n_samples=n_samples,
+                    n_channels=n_channels, stim_channel=stim_channel)
+    info['chs'] = chs
+    info._check_consistency()
+    return info, cnt_info
+
+
+def _topo_to_sphere(pos):
+    """Helper function for transforming xy-coordinates to sphere.
+    Parameters
+    ----------
+    pos : array of shape (xs, ys)
+        Coordinates to transform.
+
+    Returns
+    -------
+    coords : list
+        xyz-coordinates.
+    """
+    xs = np.array(pos)[:, 0]
+    ys = np.array(pos)[:, 1]
+
+    xs -= min(xs)  # First normalize the points.
+    ys -= min(ys)
+    xs *= (2. / max(xs))
+    ys *= (2. / max(ys))
+    xs -= 1.  # Values range from -1 to 1
+    ys -= 1.
+    coords = list()
+    for x, y in zip(xs, ys):
+        t = np.sqrt(x ** 2 + y ** 2)
+        if t > 1:  # Force the points to the surface of a sphere.
```

Comment:
consider storing with the other transformation functions or modifying existing function (cf. `mne.transforms._cartesian_to_sphere`)